Translation of the How_to_buy_en for the pt-br (#238)
monero-ecosystem/outreach-docs: monero-outreach-docs/translations/pt-br/archive/how_to_buy_pt-br.md

﻿# Como comprar Monero

> **“É fácil comprar Monero, aqui está o que você precisa saber como comprar com privacidade.” - 23 de Julho de 2020**

Reviewer: ```suggestion
**“É fácil comprar Monero, aqui está o que você precisa saber sobre como comprar com privacidade.” - 23 de Julho de 2020**

```

> Monero é uma moeda de privacidade, que é o motivo pelo qual estamos todos aqui. Por enquanto nós iremos discutir todos os caminhos para comprar Monero - incluindo métodos sem privacidade - tudo dentro da perspectiva do objetivo principal do Monero, a privacidade. A blockchain do Monero em si é privada, mas tudo o mais está sobre vigilância e irá comprometer sua privacidade quando for comprar Monero (XMR). A regra é que seu nível geral de privacidade será determinado pelas propriedades da moeda que você usa para comprá-lo. As propriedades relevantes da moeda para a compra de Monero são as seguintes:

Reviewer: ```suggestion
Monero é uma moeda de privacidade, que é o motivo pelo qual estamos todos aqui. Assim, enquanto estivermos discutindo todos os caminhos para comprar Monero - incluindo métodos sem privacidade - o faremos dentro da perspectiva do objetivo principal do Monero, a privacidade. A blockchain do Monero em si é privada, mas todo o restante está sobre vigilância e irá comprometer sua privacidade quando for comprar Monero (XMR). A regra é que seu nível geral de privacidade será determinado pelas propriedades da moeda que você usa para comprá-lo. As propriedades relevantes da moeda para a compra de Monero são as seguintes:

```

> - _fungibilidade_ - A ideia é que 1 XMR sempre será igual a outro 1 XMR não importa seu histórico e não pode ser contaminado ou discriminado. Fungibilidade pode ser forjada por políticas governamentais ou criada por tecnologia focada em privacidade, assim como o Monero.

Reviewer: ```suggestion
- _fungibilidade_ - A ideia é que 1 XMR sempre será igual a outro 1 XMR, não importa seu histórico e este não pode ser contaminado ou discriminado. Fungibilidade pode ser forjada por políticas governamentais ou criada por tecnologia focada em privacidade, assim como o Monero.

```

> - _descentralização_ - Moeda centralizada está sobre controle e vigilância. Qualquer método de pagamento centralizado para comprar Monero é fortemente assistido. Outro efeito da centralização quando se trata de comprar Monero é a tendência de altas taxas de transação.

Reviewer: ```suggestion
- _descentralização_ - Descentralização da moeda diz respeito a controle e vigilância. Qualquer método de pagamento centralizado para comprar Monero é fortemente vigiado. Outro efeito da centralização quando se trata de comprar Monero é a tendência de altas taxas de transação.

```

- _imutabilidade_ - Transações devem ser imutáveis por qualquer pessoa por qualquer motivo. Isto significa que ninguém pode reverter seus pagamentos ou congelar seus fundos. Imutabilidade descentralizada é o ponto principal da tecnologia blockchain. 

Qualquer moeda que você usar para comprar Monero terá prós e contras relacionados a essas propriedades que você precisará comparar com seu caso de uso individual e privacidade desejada.

### _Mas primeiro, duas coisas para estar seguro_
---

**“Se não são suas chaves,
não é seu Monero.”**
_-Todas as crianças legais_

> 1. Até que o XMR esteja em sua carteira ele não é seu XMR e você não será beneficiado pela privacidade do Monero. Contudo quando você comprar Monero, sempre transfira para sua carteira, para suas chaves.

Reviewer: ```suggestion
1. Até que o XMR esteja em sua carteira ele não é seu XMR e você não será beneficiado pela privacidade do Monero. Sempre que você comprar Monero, transfira-os para sua carteira, para suas chaves.

```

2. Use subendereços e não seu endereço principal quando for receber XMR. Esses são endereços descartáveis que ajudam a proteger sua privacidade. Eles também ajudam a organizar as transações recebidas já que a privacidade do remetente também está sendo protegida pelo Monero e você não saberá de outra forma quem enviou o que.

### _Comprando Monero com eletricidade_
---

> Ok, então isso é mineração de Monero, e não comprar Monero. Mas nós so queremos apontar que a mineração é a maneira mais privada de adquirir Monero e ajuda a fortalecer a rede para todos. Mineração está fora do escopo deste artigo, mas você pode aprender mais em [Minere Monero para fortalecer a rede](https://www.monerooutreach.org/praticas_recomendadas_monero/mina-para-apoiar-a-rede.html).

Reviewer: ```suggestion
Ok, então isso é mineração de Monero, e não comprar Monero. Mas nós só queremos apontar que a mineração é a maneira mais privada de adquirir Monero e ajuda a fortalecer a rede para todos. Mineração está fora do escopo deste artigo, mas você pode aprender mais em [Minere Monero para fortalecer a rede](https://www.monerooutreach.org/praticas_recomendadas_monero/mina-para-apoiar-a-rede.html).

```

### _Compre Monero com dinheiro físico_
---

> Dinheiro físico é uma das poucas moedas que é quase tão fungível quanto o Monero. A maioria das contas tem um número de série e pode ser rastreado, mas para o uso diário, dinheiro físico é praticamente fungível. Embora menos comum, trocar XMR por ouro, prata ou outra mercadoria preciosa também não é rastreável. Transações com dinheiro físico são geralmente realizadas pessoalmente, o que permite ao comprador e ao vendedor ter o controle sobre o nível de privacidade da transação. Contudo, reunião pessoal nem sempre é fácil ou livre de riscos e tende a ser geograficamente limitado. O site [localmonero.co](https://localmonero.co/) pode ajudar você a encontrar pessoas vendendo XMR e que aceitam dinheiro físico próximo a você.

Reviewer: ```suggestion
Dinheiro físico é uma das poucas moedas que é quase tão fungível quanto o Monero. A maioria das notas possuem um número de série e pode ser rastreada, mas para o uso diário, dinheiro físico é praticamente fungível. Embora menos comum, trocar XMR por ouro, prata ou outra mercadoria preciosa também não é rastreável. Transações com dinheiro físico são geralmente realizadas pessoalmente, o que permite ao comprador e ao vendedor ter o controle sobre o nível de privacidade da transação. Contudo, encontros presenciais nem sempre são fáceis ou livres de riscos e tendem a ser geograficamente limitados. O site [localmonero.co](https://localmonero.co/) pode ajudar você a encontrar pessoas próximas a você que estejam vendendo XMR e que aceitem dinheiro físico .

```

_**Prós**_

- Privado/Fungível
- Decentralizado

> - Custo de transação baixo/gratis

Reviewer: ```suggestion
- Custo de transação baixo/grátis

```

_**Contras**_

- Inconveniente

### _Comprar Monero com um Instrumento Portador_
---

Instrumento portador é apenas um termo sofisticado que significa que se estiver no seu bolso, você provou que é seu. Bilhetes de loteria premiado e alguns títulos são instrumento portador, mas os que vão ajudar você a comprar Monero serão Ordens de Pagamento do Serviço Postal dos Estados Unidos (USPS) e em uma extensão muito menor, cheques administrativos.

Bisq suporta Ordens de Pagamento USPS. Disponível apenas nos Estados Unidos, você pode comprar $3,000 em valor de ordens de pagamento por dia nos EUA sem identificação. Além disso pode ser comprado como dinheiro físico para aumentar ainda mais sua privacidade. Visite [bisq.wiki/US_Postal_Money_Order](https://bisq.wiki/US_Postal_Money_Order) para mais informações sobre como isto funciona e como se manter seguro. No momento da escrita deste artigo, uma ordem de pagamento está custando entre $1,25 ou $1,70 dólares dependendo da quantia.

> Um cheque administrativos também prover um grau de privacidade pois se trata literalmente de um cheque bancário que é sacado da conta do próprio banco e não da sua. Você estará em um cenário KYC (sigla em inglês para Conheça-Seu-Cliente) com o banco em que você comprou o cheque, mas até certa quantia você estará livre de ser reportado ao AML (sigla em inglês para Anti-Lavagem-de-Dinheiro). Depois de tudo isso, o benefício seria um anonimato relativo contra qualquer coisa menor que uma intimidação.

Reviewer: ```suggestion
Um cheque administrativo também provê um grau de privacidade pois se trata literalmente de um cheque bancário que é sacado da conta do próprio banco e não da sua. Você estará em um cenário KYC (sigla em inglês para Conheça-Seu-Cliente) com o banco em que você comprou o cheque, mas até certa quantia você estará livre de ser reportado ao AML (sigla em inglês para Prevenção à Lavagem de Dinheiro). Depois de tudo isso, o benefício seria um anonimato relativo contra qualquer coisa menor que uma intimidação.

```

_**Prós**_

- Anônimo
- Baixo custo

_**Contras**_

- Inconveniente

### _Compre Monero em um caixa eletrônico de criptomoedas (ATM)_
---

> Vários caixas eletrônicos de Monero aceitam dinheiro físico e geralmente pequenas transações podem ser concluídas anonimamente. Os proprietários de ATM’s podem ter políticas ligeiramente diferentes sobre isso, mas regulações de Conheça-Seu-Cliente/Anti-Lavagem-de-Dinheiro (KYC/AML) surgem invariavelmente a partir de determinada quantia transacionada e os dados da sua transação seriam armazenados pelo proprietário do ATM não importa o valor. O site coinatmradar.com fornece a localização de ATM’s Monero

Reviewer: ```suggestion
Vários caixas eletrônicos de Monero aceitam dinheiro físico e geralmente pequenas transações podem ser concluídas anonimamente. Os proprietários de ATM’s podem ter políticas ligeiramente diferentes sobre isso, mas regulações de Conheça-Seu-Cliente/Prevenção à Lavagem de Dinheiro (KYC/AML) surgem invariavelmente a partir de determinada quantia transacionada e os dados da sua transação seriam armazenados pelo proprietário do ATM, não importando o valor. O site coinatmradar.com fornece a localização de ATM’s Monero

```

_**Prós**_

- Anônimo para pequenas quantias

_**Contras**_

- Alto custo
- Limitado geograficamente
- Dados da transação são armazenados

### _Compre Monero com Cripto_
---

> Se você já tem criptomoedas e gostaria de trocar por Monero você tem bastante opções. Bitcoin-por-Monero (BTC-XMR) é de longe o mais comum par de troca  e rampa de entrada que você irá encontrar, mas a Binance mantém Ethereum-por-Monero (ETH-XMR), e com certeza outros pares surgirão. Se você tem outro tipo de criptomoeda, você precisará primeiro convertê-la para uma que você pode trocar diretamente por XMR, o que frequentemente pode ser feito usando a mesma corretora. Há três tipos principais de corretoras que você pode usar.

Reviewer: ```suggestion
Se você já tem criptomoedas e gostaria de trocar por Monero você tem bastante opções. Bitcoin-por-Monero (BTC-XMR) é de longe o par de troca e porta de entrada mais comum que você irá encontrar, mas a Binance mantém Ethereum-por-Monero (ETH-XMR), e com certeza outros pares surgirão. Se você tem outro tipo de criptomoeda, você precisará primeiro convertê-la para uma que você pode trocar diretamente por XMR, o que frequentemente pode ser feito usando a mesma corretora. Há três tipos principais de corretoras que você pode usar.

```

#### P2P – Corretora Pessoa para Pessoa 

> Você pode depositar seu BTC em uma corretora privada e descentralizada como a [bisq.network](https://bisq.network/) para mais privacidade. Corretoras P2P são descentralizadas e permitem as pessoas trocar cripto sem a necessidade uma terceira parte. 

Reviewer: ```suggestion
Você pode depositar seu BTC em uma corretora privada e descentralizada como a [bisq.network](https://bisq.network/) para mais privacidade. Corretoras P2P são descentralizadas e permitem às pessoas trocarem cripto sem a necessidade em terceiro ou intermediário. 

```

#### Corretora Sem-KYC 

> Corretoras como [morphtoken.com](https://www.morphtoken.com/) e [tradeogre.com](https://tradeogre.com/) são centralizadas mas não exigem KYC porque elas não fornecem conversão para moeda Fiat.

Reviewer: ```suggestion
Corretoras como [morphtoken.com](https://www.morphtoken.com/) e [tradeogre.com](https://tradeogre.com/) são centralizadas, mas não exigem KYC porque elas não fornecem conversão para moeda fiduciária.

```

[cakewallet.com](https://cakewallet.com/) tem uma característica que permite a você trocar Bitcoin, Ethereum, Bitcoin Cash, Litecoin e Dash por Monero na própria carteira. Isto é bom para pessoas que desejam trocar Monero, mas não querem criar outra conta em corretora. 

#### Corretora centralizada com KYC

> Por último, você poderia também depositar entre alguma corretora convencional que oferece negociação no par BTC-XMR. É importante lembrar que até que você transfira seu XMR para sua carteira, ele não estará seguro e nem privado.

Reviewer: ```suggestion
Por último, você poderia também depositar em alguma corretora convencional que oferece negociação no par BTC-XMR. É importante lembrar que até que você transfira seu XMR para sua carteira, ele não estará seguro e nem será privado.

```

_**Prós**_

- Prático

_**Contras**_

- Não privado

### _Comprar Monero com Cartão de Crédito/Débito_
---

> Uma transação de cartão de crédito de qualquer tipo é completamente centralizada e não é nada privado. Isso não deveria ser uma surpresa, mas é pior do que você pode imaginar. Nós assumimos que as companhias de cartão de crédito tem todas as informações sobre nossas transações mas nós geralmente não pensamos que os processadores de pagamento e os proprietários de rede também possuem esses dados. Com a regularidade das violações de dados dos dias atuais, uma transação com cartão de crédito tem uma área de superfície de exposição muito grande. Adicionalmente é comum para essas transações atravessar fronteiras sem o seu conhecimento. Você deve estar fazendo compras em uma loja local usando um cartão de banco local, mas é provável que seus dados estão sendo enviados e armazenados através dos Estados Unidos. Com aquela ressalva, usando um cartão de crédito com acesso direto ao Monero é a forma mais rápida e simples para comprar XMR e nossa página [Recursos para Investidores](https://www.monerooutreach.org/investidores/) disponibiliza uma lista de corretoras onde você pode comprar Monero diretamente.

Reviewer: ```suggestion
Uma transação de cartão de crédito de qualquer tipo é completamente centralizada e não é nada privado. Isso não deveria ser uma surpresa, mas é pior do que você pode imaginar. Nós assumimos que as companhias de cartão de crédito tem todas as informações sobre nossas transações, mas nós geralmente não pensamos que os processadores de pagamento e os proprietários de rede também possuem esses dados. Com a regularidade das violações de dados dos dias atuais, uma transação com cartão de crédito tem uma área de superfície de exposição muito grande. Adicionalmente, é comum para essas transações atravessar fronteiras sem o seu conhecimento. Você deve estar fazendo compras em uma loja local usando um cartão de banco local, mas é provável que seus dados estão sendo enviados e armazenados através dos Estados Unidos. Com essa ressalva, usar um cartão de crédito para acesso direto ao Monero é a forma mais rápida e simples para comprar XMR e nossa página [Recursos para Investidores](https://www.monerooutreach.org/investidores/) disponibiliza uma lista de corretoras onde você pode comprar Monero diretamente.

```

Cartão de crédito pré-pago pode fornecer anonimato contanto que não haja outro metadado para conectar você àquele cartão. Se você comprar um cartão pré-pago com dinheiro e não usá-lo para outra compra, você terá um nível razoável de privacidade.

_**Prós**_

- Prático

_**Contras**_

- Não privado
- Centralizado
- Alto risco de violação

### _Comprar Monero com Transferência bancária_
---

> Transferências Bancárias são mais comumente usadas para transações OTC (Sobre-o-Balcão) maiores. Vale a pena observar que pagamentos eletrônicos são diferentes de crédito/débito até se for o mesmo banco oferecendo todos os três serviços, pois eles são realizados de forma diferente. Transações de crédito/débito passam por alguns processos e redes como qualquer outro cartão de crédito. Similarmente, qualquer serviço de transferência de email tende a percorrer uma rede de terceiro. Mas transferências conta a conta e transferências eletrônicas formais seguem um caminho diferente. Uma transferência de conta interna é feita dentro da rede desse banco o que significa apenas que os dados dessa transação existem em menos lugares e seriam processados instantaneamente. Uma transferência eletrônica contudo introduz novos grupos de vigilância, mais notavelmente no banco destinatário e nos bancos intermediários, mas também SWIFT (Sociedade Internacional de Telecomunicação Financeira Entre Bancos). Se sua conexão está indo de dentro ou de fora dos Estados Unidos também passará através do Escritório de Controle de Ativos Estrangeiros e Fedwire.

Reviewer: ```suggestion
Transferências Bancárias são mais comumente usadas para transações OTC (Sobre-o-Balcão) maiores. Vale a pena observar que pagamentos eletrônicos são diferentes de crédito/débito até se for o mesmo banco oferecendo todos os três serviços, pois eles são realizados de forma diferente. Transações de crédito/débito passam por alguns processos e redes como qualquer outro cartão de crédito. Similarmente, qualquer serviço de transferência de email tende a percorrer uma rede de terceiro. Mas transferências conta a conta e transferências eletrônicas formais seguem um caminho diferente. Uma transferência de conta interna é feita dentro da rede desse banco, o que significa apenas que os dados dessa transação existem em menos lugares e seriam processados instantaneamente. Uma transferência eletrônica, contudo, introduz novos grupos de vigilância, mais notavelmente no banco destinatário e nos bancos intermediários, mas também SWIFT (Sociedade Internacional de Telecomunicação Financeira Entre Bancos). Se sua conexão está indo de dentro ou de fora dos Estados Unidos ela também passará através do Escritório de Controle de Ativos Estrangeiros e FedWire.

```

_**Prós**_

- Adequado para grandes quantias

_**Contras**_

- Não privado
- Centralizado
- Caro e Inconveniente

### _Comprar Monero com Paypal_
---

> Paypal, Apple Pay, Google Pay, Venmo ou Western Union são tipos de serviços que não são privados e podem ser caros. Estes serviços de transferência de dinheiro também possuem seus próprios Termos e Políticas de Serviço e pode lhe trazer riscos adicionais. Em particular com o Paypal, vendedores de XMR precisam ser cuidadosos  já que as transações podem ser revertidas e as políticas do Paypal tendem a ficar do lado do comprador em caso de disputa. Em um exemplo de golpe, Alice aceita comprar 1 XMR por X dólares de Bob Alice envia X dólares para Bob via Paypal, e então Bob envia para Alice 1 XMR. Alice abre uma disputa no Paypal e é reembolsada em X dólares, e ela mantém o Monero que comprou.

Reviewer: ```suggestion
Paypal, Apple Pay, Google Pay, Venmo ou Western Union são tipos de serviços que não são privados e podem ser caros. Estes serviços de transferência de dinheiro também possuem seus próprios Termos e Políticas de Serviço e podem lhe trazer riscos adicionais. Particularmente com o Paypal, vendedores de XMR precisam ser cuidadosos já que as transações podem ser revertidas e as políticas do Paypal tendem a ficar do lado do comprador em caso de disputa. Em um exemplo de golpe, Alice aceita comprar 1 XMR por X dólares de Bob. Alice envia X dólares para Bob via Paypal, e então Bob envia para Alice 1 XMR. Alice abre uma disputa no Paypal e é reembolsada em X dólares, e ela mantém o Monero que comprou.

```
